systemtest/src/test/java/io/strimzi/systemtest/log/LoggingChangeST.java

```diff
@@ -349,6 +352,111 @@ void testDynamicallySetEOloggingLevels() throws InterruptedException {
+        // wait some time and check whether logs after this time contain anything
+        Thread.sleep(LOGGING_RELOADING_INTERVAL * 2);
+
+        LOGGER.info("Asserting if log will contain some records");
+        assertThat(StUtils.getLogFromPodByTime(bridgePodName, KafkaBridgeResources.deploymentName(CLUSTER_NAME), "30s"), is(not(emptyString())));
+
+        ConfigMap configMapBridge = new ConfigMapBuilder()
+                .withNewMetadata()
+                .withName("external-configmap-bridge")
+                .withNamespace(NAMESPACE)
+                .endMetadata()
+                .withData(Collections.singletonMap("log4j2.properties",
+                        "name = BridgeConfig\n" +
+                                "\n" +
+                                "appender.console.type = Console\n" +
+                                "appender.console.name = STDOUT\n" +
+                                "appender.console.layout.type = PatternLayout\n" +
+                                "appender.console.layout.pattern = [%d] %-5p <%-12.12c{1}:%L> [%-12.12t] %m%n\n" +
+                                "\n" +
+                                "rootLogger.level = OFF\n" +
+                                "rootLogger.appenderRefs = console\n" +
+                                "rootLogger.appenderRef.console.ref = STDOUT\n" +
+                                "rootLogger.additivity = false\n" +
+                                "\n" +
+                                "logger.bridge.name = io.strimzi.kafka.bridge\n" +
+                                "logger.bridge.level = OFF\n" +
+                                "logger.bridge.appenderRefs = console\n" +
+                                "logger.bridge.appenderRef.console.ref = STDOUT\n" +
+                                "logger.bridge.additivity = false\n" +
+                                "\n" +
+                                "# HTTP OpenAPI specific logging levels (default is INFO)\n" +
+                                "# Logging healthy and ready endpoints is very verbose because of Kubernetes health checking.\n" +
+                                "logger.healthy.name = http.openapi.operation.healthy\n" +
+                                "logger.healthy.level = OFF\n" +
+                                "logger.ready.name = http.openapi.operation.ready\n" +
+                                "logger.ready.level = OFF"))
+                .build();
+
+        kubeClient().getClient().configMaps().inNamespace(NAMESPACE).createOrReplace(configMapBridge);
+
+        ExternalLogging bridgeXternalLogging = new ExternalLoggingBuilder()
+                .withName("external-configmap-bridge")
+                .build();
+
+        LOGGER.info("Setting log level of Bridge to OFF - records should not appear in the log");
+        // change to the external logging
+        KafkaBridgeResource.replaceBridgeResource(CLUSTER_NAME, bridz -> {
```

Comment:
bridz?

Reply:
Yeah, it is a prevention of using test-scope-var and lambda-scope-var with the same name. I can rename it if you wish so.

Reply:
is that a czech word? :-)

Reply:
Kinda :)